app/editor/src/features/navbar/NavBar.tsx

```diff
@@ -1,38 +1,43 @@
-import React from 'react';
-import { useLocation } from 'react-router-dom';
-import { Claim } from 'tno-core';
+import React, { useState } from 'react';
+import { Claim, NavBarGroup, NavBarItem, Show } from 'tno-core';
 import { Row } from 'tno-core/dist/components/flex/row';
-import { NavBarGroup, NavBarItem } from 'tno-core/dist/components/navbar';
 
 /**
  * The navigation bar that is used throughout the TNO editor application. Add or remove navigation bar items here.
  */
 export const NavBar: React.FC = () => {
-  const location = useLocation();
+  const [activeHover, setActiveHover] = useState<'admin' | 'editor' | ''>('');
```

Comment:
What's the value of hardcoded values here?  Why not just use string?

Reply:
I'm wondering how we can convert this to a generic component, but doesn't need to be done now.